store/sqlstore/post_store.go

```diff
@@ -1606,6 +1607,8 @@ func (s *SqlPostStore) determineMaxPostSize() int {
 				table_name = 'posts'
 			AND	column_name = 'message'
 		`); err != nil {
+			// REVIEW: I think this should be returned as an error and we should remove the error log here, but the whole code is not prepared to handle an error here.
```

Comment:
We have to decide on some action if the SQL query fails. I don't think we should fail the top-level action because of this as this is not a must-have thing for that to pass. So I think logging should be fine.

I think we should change mlog.Error to mlog.Warn, as this is not an actionable error, and nothing really fails. But more like a warning.

Reply:
Changed